functions-and-scope.js

```diff
@@ -14,6 +15,17 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // * Hoe zorgt ik ervoor dat wanneer ik een cijfer tegenkom die aan de conditie voldoet, ik dit ergens kan bijhouden?
 // Log het antwoord in de terminal.
 
+let cumLaudeNumbers = 0;
+for (let i=0; i < grades.length; i++){
+    //Let is variabel, veranderd. De index is 0, we kijken naar de lengte van de cijfers, als die korter is dan gaan we een stap omhoog.
+    if (grades[i] >= 8) {
+        // We willen dat hij Cumlaude geeft als hij gelijk aan het cijfer 8 is of hoger.
+        cumLaudeNumbers = cumLaudeNumbers +1;
```

Comment:
Dit zou je nog wat kunnen versimpelen
```suggestion
        cumLaudeNumbers++;
```